### PR TITLE
Add meta example for ChiSquare Distance

### PR DESCRIPTION
**Pull request by syashakash**
Refer #3555 
All tests passed.

### CODE REVIEW

**Comment by syashakash** [comment]
unit-SGObject encounters Segmentation Fault, but the test works fine locally. I would get back to this later this week. I can't figure out what could possibly have gone wrong!!

**Comment by syashakash** [comment]
Added data file for this example.

**Comment by syashakash** [comment]
Added data revision to this PR.

**Comment by karlnapf** [comment]
Thanks!
We will need to wait with merging this, as there currently is a problem with the internal tests that needs fixing first (and likely update of the data file).

**Comment by syashakash** [comment]
Agreed :)

**Comment by vigsterkr** [comment]
@syashakash could you rebase this with the latest develop as the integration test is back.

**Comment by syashakash** [comment]
Ok, I see that the integration tests have failed! I would see how to solve this.

File: examples/meta/src/distance/chisquare.sg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+CSVFile f_feats_a("../../data/fm_train_real.dat")
```
Comment (karlnapf) [inline examples/meta/src/distance/chisquare.sg]: pls rename the file to chi_square

**Comment by syashakash** [comment]
@karlnapf @vigsterkr this example fails the new binary kernel SVM integration tests for ChiSquareDistance. The example's code is similar to that of other `distance` examples. I can't understand where the code fails as it does what it is supposed to do.

**Comment by karlnapf** [comment]
The integration test fails if the listing produces different data than the one in the reference data. See the examples and the development readme

**Comment by karlnapf** [comment]
Shall we get this merged?

**Comment by syashakash** [comment]
@karlnapf the integration tests fail locally. Moreover, the data file needs another update, and this PR has wrong file.

**Comment by karlnapf** [comment]
Sure, wanna go ahead with that?

**Comment by karlnapf** [comment]
There now also is a merge conflict, so you will need to update this PR

**Pull request closed without merging**